docs/tutorials/python/file.md

```diff
@@ -0,0 +1,229 @@
+# Files in Synapse
+Synapse files can be created by uploading content from your local computer or linking to digital files on the web.
+
+Files in Synapse always have a “parent”, which could be a project or a folder. You can organize collections of files into folders and sub-folders, just as you would on your local computer.
+
+[Read more](../../explanations/domain_models_of_synapse.md#files)
+
+
+This tutorial will follow a [Flattened Data Layout](../../explanations/structuring_your_project.md#flattened-data-layout-example). With this example layout:
+```
+.
+├── experiment_notes
+│   ├── notes_2022
+│   │   ├── fileA.txt
+│   │   └── fileB.txt
+│   └── notes_2023
+│       ├── fileC.txt
+│       └── fileD.txt
+├── single_cell_RNAseq_batch_1
+│   ├── SRR12345678_R1.fastq.gz
+│   └── SRR12345678_R2.fastq.gz
+└── single_cell_RNAseq_batch_2
+    ├── SRR12345678_R1.fastq.gz
+    └── SRR12345678_R2.fastq.gz
+```
+
+## Tutorial Purpose
+In this tutorial you will:
+
+1. Upload several files to Synapse
+1. Print stored attributes about your files
+1. List all Folders and Files within my project
+
+
+## Prerequisites
+* Make sure that you have completed the [Folder](./folder.md) tutorial.
+* The tutorial assumes you have a number of files ready to upload. If you do not, create a test or dummy file.
```

Comment:
Looking for input what the best way to mention this part is. Perhaps we should create a python gist that creates the above file structure?

Reply:
Hmm, we could provide some dummy files. There's probably a "fake" small biology dataset we can download.

Reply:
Good idea. I am going to include some of the sample files I was using during the tutorial. I just echo'd some sample data into files and named them